Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -316,4 +316,77 @@ Cycript tricks:
 
 http://iphonedevwiki.net/index.php/Cycript_Tricks
 
-<!-- TODO [Develop section on Frida] -->
+#### [Frida](https://www.frida.re/docs/home/) 
+
+##### The documentation of Frida describes[how to install and verify the installation](https://www.frida.re/docs/installation/)
+
+###### Without Jailbreak
+This is the most powerful setup, as it lets you instrument system services and apps with very little effort.
+
+###### Setting up your iOS device
+
+Start Cydia and add Frida’s repository by going to **Manage -> Sources -> Edit -> Add and enter https://build.frida.re**. You should now be able to find and install the Frida package which lets Frida inject JavaScript into apps running on your iOS device. This happens over USB, so you will need to have your USB cable handy, though there’s no need to plug it in just yet.
```

Comment:
This is all just copy paste from here https://github.com/frida/frida-website/blob/master/_docs/ios.md

Please just link to it and don't copy the content in the MSTG.